bin/pycbc_optimize_snr

```diff
@@ -101,6 +106,11 @@ def compute_network_snr(v, *argv):
         argv = argv[0]
     nsnr, _ = compute_network_snr_core(v, *argv)
     return -nsnr
+ 
+def compute_network_snr_pso(v, *argv, **kwargs):
+    argv = kwargs['args']
+    nsnr_array = [-compute_network_snr_core(v_i, *argv)[0] for v_i in v]
```

Comment:
This was a numpy array before, now it's not, would that break anything?

Reply:
I've tested it on all 3 optimizers and it successfully completes the optimization. The only error I get is `KeyError: "Unable to open object (object 'mc_area_args' doesn't exist)"` but this was also the case from way back when I did the earlier changes.

Reply:
You can easily make it a numpy array like so,
```
numpy.array([list comprehension])
```

Reply:
Is it safer to convert it into a numpy array?

Reply:
It may be a bit more memory-efficient if `v` is a large integer, but I doubt it would make a practical difference.